packages/pharos/src/components/progress-bar/pharos-progress-bar.scss

```
@@ -31,3 +31,25 @@
 
   color: var(--pharos-color-text-40);
 }
+
+.progress-bar-indeterminate {
```

Nit: `indeterminate` feels like a modifier of progress bar, so using `--` better reflects that in BEM-style CSS (needs adjustment in the component code as well, if changing):

```suggestion
.progress-bar--indeterminate {
```